src/coreclr/inc/corinfo.h

```diff
@@ -2009,6 +2009,9 @@ class ICorStaticInfo
     // Quick check whether the method is a jit intrinsic. Returns the same value as getMethodAttribs(ftn) & CORINFO_FLG_INTRINSIC, except faster.
     virtual bool isIntrinsic(CORINFO_METHOD_HANDLE ftn) = 0;
 
+    // Quick check whether the method is a pinvoke
+    virtual bool isPinvoke(CORINFO_METHOD_HANDLE ftn) = 0;
```

Comment:
I believe this question can be answered using `getMethodAttribs()` and check for the `CORINFO_FLG_PINVOKE`.

Reply:
> I believe this question can be answered using `getMethodAttribs()` and check for `CORINFO_FLG_PINVOKE`.

`getMethodAttribs` does a lot of things we don't need here and since we want to make prescan as quick as possible I introduced a separate API

Reply:
Isn't going to be called somewhere anyways? I can't imagine the JIT hasn't or isn't about to call that function anyways.